<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>681</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Flight Strip Manager</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttonAdd">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonReset">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="CustomListWidget" name="listPend">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="toolTipDuration">
     <number>1</number>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::MoveAction</enum>
    </property>
   </widget>
   <widget class="CustomListWidget" name="listPush">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>111</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::MoveAction</enum>
    </property>
   </widget>
   <widget class="CustomListWidget" name="listDepa">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>111</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::MoveAction</enum>
    </property>
   </widget>
   <widget class="CustomLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
    <property name="placeholderText">
     <string>Callsign</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="LCDTimer" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Left click:&lt;/span&gt; start/stop timer&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Right click:&lt;/span&gt; cycle preset time&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Scroll:&lt;/span&gt; precise control&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuSet_opacity">
     <property name="title">
      <string>Set opacity</string>
     </property>
     <addaction name="actionOpac_Curr"/>
     <addaction name="separator"/>
     <addaction name="actionOpac_Incr"/>
     <addaction name="actionOpac_Decr"/>
     <addaction name="actionOpac_Rest"/>
    </widget>
    <widget class="QMenu" name="menuTimer_format">
     <property name="title">
      <string>Timer format</string>
     </property>
     <addaction name="actionSec"/>
     <addaction name="actionMinSec"/>
    </widget>
    <addaction name="menuSet_opacity"/>
    <addaction name="actionStay_on_top"/>
    <addaction name="menuTimer_format"/>
    <addaction name="separator"/>
    <addaction name="actionStatistics"/>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionReset"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMain"/>
  </widget>
  <action name="actionStay_on_top">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Stay on top</string>
   </property>
   <property name="shortcut">
    <string>Alt+Return</string>
   </property>
  </action>
  <action name="actionStatistics">
   <property name="text">
    <string>Statistics</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpac_Incr">
   <property name="text">
    <string>Increase</string>
   </property>
   <property name="shortcut">
    <string>Alt+=</string>
   </property>
  </action>
  <action name="actionOpac_Decr">
   <property name="text">
    <string>Decrease</string>
   </property>
   <property name="shortcut">
    <string>Alt+-</string>
   </property>
  </action>
  <action name="actionOpac_Rest">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="shortcut">
    <string>Alt+Backspace</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset lists</string>
   </property>
  </action>
  <action name="actionOpac_Curr">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Current: 100%</string>
   </property>
  </action>
  <action name="actionSec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Seconds</string>
   </property>
  </action>
  <action name="actionMinSec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Min:Sec</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>CustomListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">listwidget.h</header>
  </customwidget>
  <customwidget>
   <class>LCDTimer</class>
   <extends>QLCDNumber</extends>
   <header location="global">timer.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>buttonAdd</tabstop>
  <tabstop>listPend</tabstop>
  <tabstop>listPush</tabstop>
  <tabstop>listDepa</tabstop>
  <tabstop>buttonReset</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonAdd</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addFlight()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonReset</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>resetLists()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>buttonAdd</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addFlight()</slot>
  <slot>resetLists()</slot>
  <slot>setStatusClr()</slot>
  <slot>setStatusDep()</slot>
  <slot>clickMenu()</slot>
 </slots>
</ui>
